bchrpc/proxy/.gitignore

```diff
@@ -0,0 +1 @@
+gw
```

Comment:
Since this file is nested in the repo, we should put this in the root .gitignore and remove this one.

Reply:
Seems you missed a few comments from the previous review... most of my original comments had no response.

Reply:
> Seems you missed a few comments from the previous review... most of my original comments had no response.

Correct, sorry about this.  It seems github had them collapsed and I probably needed to click "Show X more comments" to expand..  working on them now.

Reply:
> Overall this looks great. Have we done IBD tests on testnet/mainnet? Just want to make sure that has been run on the latest code before we merge. =)

I will do this, hopefully will be completed by the end of the week.